Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -531,10 +531,14 @@ const ScrollView = React.createClass({
         // On Android wrap the ScrollView with a AndroidSwipeRefreshLayout.
         // Since the ScrollView is wrapped add the style props to the
         // AndroidSwipeRefreshLayout and use flex: 1 for the ScrollView.
+        // Note: we should only apply prop.style on the wrapper
+        // but the ScrollView still needs the baseStyle to be scrollable
+        const scrollViewProps = { ...props, style: baseStyle };
+
         return React.cloneElement(
           refreshControl,
           {style: props.style},
-          <ScrollViewClass {...props} ref={this._setScrollViewRef}>
+          <ScrollViewClass {...scrollViewProps} ref={this._setScrollViewRef}>
```

Comment:
Let's avoid the extra props copy and just pass the style in jsx directly. Something like:
``` js
<Component {...props} style={...} />
```